third_party/nvidia_sdk_download/fetch_cuda_toolkit.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+# Copyright 2022 The IREE Authors
```

Comment:
Wait now we have a thing under the IREE license that's in a directory with a different license... :see_no_evil:

This is just shelling out to parse_redist, so it seems not that hard to locate it elsewhere and find the file (separately it would be nice to not have to invoke another process there, but I think it's not worth rewriting the script to avoid it)